ONSdigital/perkin
# Reinstating SSL verification of the RRM receipt host.

This feature can be switched on or off using the NO_SSL_VERIFICATION=true|false feature switch (until we're happy that it's stable, then we'll take the switch out).

How to test:
pass "-e NO_SSL_VERIFICATION=true|false" (or no flag or variable at all for false) and check that the SSL connection succeeds. You should see the configured value printed out on startup.
